src/doge/wow.py

```diff
@@ -149,18 +170,19 @@ def easter_dates():
 # If you see this and use a light terminal, a pull request with a set that
 # works well on a light terminal would be awesome.
 COLORS = DogeDeque(
-    23, 24, 25, 26, 27, 29, 30, 31, 32, 33, 35, 36, 37, 38, 39, 41, 42, 43,
-    44, 45, 47, 48, 49, 50, 51, 58, 59, 63, 64, 65, 66, 67, 68, 69, 70, 71,
-    72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 94,
-    95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109,
-    110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123,
-    130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143,
-    144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157,
-    158, 159, 162, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176,
-    177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190,
-    191, 192, 193, 194, 195, 197, 202, 203, 204, 205, 206, 207, 208, 209,
-    210, 211, 212, 213, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223,
-    224, 225, 226, 227, 228
+    *(
+        int(x)
+        for x in """
+        23 24 25 26 27 29 30 31 32 33 35 36 37 38 39 41 42 43 44 45 47 48 49 50
+        51 58 59 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83
+        84 85 86 87 88 94 95 96 97 98 99 100 101 102 103 104 105 106 107 108 109
+        110 111 112 113 114 115 116 117 118 119 120 121 122 123 130 131 132 133
+        134 135 136 137 138 139 140 141 142 143 144 145 146 147 148 149 150 151
+        152 153 154 155 156 157 158 159 162 166 167 168 169 170 171 172 173 174
+        175 176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191 192
+        193 194 195 197 202 203 204 205 206 207 208 209 210 211 212 213 214 215
+        216 217 218 219 220 221 222 223 224 225 226 227 228""".split()
```

Comment:
What fresh hell is this? :sweat_smile:  Why is this preferred by the tools?

Reply:
Haha, it's not preferred by the tools... The tools want to put one word or one number per line. Using `black` or `ruff format` blows up the line count from 460 lines to 1841. This is just my attempt at keeping the line count down while using 100% auto-formatting.

If it's too unreadable, I should probably put the data into a separate JSON file or something. 😄

Reply:
Ah, fine then.  It's not like it ever really changes, haha.  It just struck me as quite odd. :sweat_smile:

Reply:
You're right, it is a bit odd...  I will add a comment to clarify.